installation/windows.md

```diff
@@ -5,27 +5,36 @@ title: Windows
 
 {% include base.html %}
 
-# openHAB on Windows
+# openHAB 2 on Windows
 
 The following instructions will guide you through the process of setting up openHAB 2.
 
 ## Prerequisites
 
 First, be sure to check that your system has the necessary [prerequisites]({{base}}/installation/index.html#prerequisites).
 
+Add the JAVA_HOME System Environment Variable in Windows: Navigate to
+`Control Panel\System and Security\System\Advanced system settings\Environment variables\System variables` and create a New System Variable based on your Java installation directory:
```

Comment:
To stay in sync with other parts I'd suggest to put JAVA_HOME in ``.
The path through the Windows menu would be better described by -> or → or ➡